.gitignore

```diff
@@ -1,15 +1,31 @@
 sandbox/
 example/data/*
+
+# Bay area data
 bay_area_urbansim/data/*
 bay_area_urbansim/output/*
 bay_area/data/*
 bay_area/output/*
+bay_area/tmp/*
+bay_area/pilates/*
+
 *.h5
 .ipynb_checkpoints
 .coverage*
 .pytest_cache
 .vagrant
 
+#Austin data
+austin/Accessibilities.ipynb
+austin/mode_choice_calibration.ipynb
+austin/new_calibration_probaflow.ipynb
+austin/validation_results.ipynb
+austin/data/*
+austin/output/*
+austin/tmp/*
+austin/pilates/*
+!austin/output/activitysim.log
+!austin/data/land_use.csv
```

Comment:
is there a reason why you're suggesting we store the main land_use.csv in the repo?